shared/src/commonMain/sqldelight/database/plant.sq

```diff
@@ -0,0 +1,39 @@
+import kotlin.Boolean;
+
+CREATE TABLE favoritePlant (
```

Comment:
It's all good for now, but I think we won't need all these fields (try to keep db small) :) just the ID of the plant maybe a userId (it is optional, just an idea if we want to add login). If the db has the plant id it could mean that the plant is favorited by user. So If you have time, you can modify it like this

Reply:
Database is modified to contain the id's only :)

Reply:
yes you are right